credits.html

```diff
@@ -0,0 +1,40 @@
+<html>
+<head>
+	<title>
+		Left Shark | I do what I want
+	</title>
+	<link rel="stylesheet" type="text/css" href="css/styles.css">
+	<link href='https://fonts.googleapis.com/css?family=Delius+Unicase' rel='stylesheet' type='text/css'>
+	<link href='https://fonts.googleapis.com/css?family=Montserrat' rel='stylesheet' type='text/css'>
+</head>
+<body>
+
+
+	<img src="images/profile.svg">
+
+	<h1>Left Shark</h1>
+
+	<h2>Dance Like Nobody's Watching</h2>
+
+	<nav>
+		<a href="index.html">go left</a>
+		<a href="resume.html">other left</a>
+		<!--  <a href="credits.html">credits</a> -->
+	</nav>
+
+		<section>
```

Comment:
:hammer: Your indentation is off here. `<section>` should line up with `</nav>` above.